### ISSUE TITLE
Bad Cases: Some short numbers can be annotated

### ISSUE THREAD

**Issue by ShomyLiu** [dbpedia-spotlight/dbpedia-spotlight-model#18]
Hi,
I found that some words such as  "02" "-5" and so on can be annotated to some entities, which is very strange. It may be the model that caused the question, which I used in 'en_2.2/'

For example
```
02: http://dbpedia.org/resource/Digimon_Adventure_02
-5: http://dbpedia.org/resource/Straight-five_engine
```

**Comment by beyzayaman** [comment]
Hi,


Are you sending a file as data in your curl? This might cause a problem. You should send your text url encoded like this:


--data-urlencode "text=Put your text here!"



________________________________

**Comment by ShomyLiu** [comment]
@beyzayaman  
Yes, What I mean is that if some words in a text are numbers, they would be annotate to some entities.
![image](https://user-images.githubusercontent.com/10215945/32814147-b1f509f4-c9e8-11e7-9883-d892d91565bc.png)

**Comment by ANSHIT175** [comment]
Hi, I am a GSoC 2026 applicant for DBpedia.

I have submitted a proposal on building a Natural Language to SPARQL system and started working on a prototype:
https://github.com/ANSHIT175/nl-to-sparql-dbpedia

I would like to contribute to DBpedia. Could you please suggest a beginner-friendly issue or area where I can help?

Thank you!
